### ISSUE TITLE
In JS, importing JSON should not give an error

### ISSUE THREAD

**Issue by sandersn** [Microsoft/TypeScript#23712]
We don't have to give a good type, but there should not be an error.

Search terms: import require json

```js
const version = require('./package.json').version
```

**Expected behavior:**
No error and maybe even `version: string`

**Actual behavior:**
Error and `version: any`.

**Comment by mhegazy** [comment]
Duplicate of #7071. Should be fixed by https://github.com/Microsoft/TypeScript/pull/22167

**Comment by sandersn** [comment]
OK, I updated #7071's title to be searchable.
